packages/cc/src/lib/_Types.ts

```diff
@@ -1670,6 +1670,22 @@ export enum ZWaveProtocolCommand {
 	SmartStartInclusionRequest = 0x28,
 }
 
+export enum ZWaveLRProtocolCommand {
+	NOP = 0x00,
+
+	// BUGBUG: all defined above, can they be shared?
```

Comment:
These have the same # assignments, but are in a separate namespace. Add the missing ones to the list above, or leave these separate (or maybe just reference `Foo = ZWaveProtocolCommand.Foo.value` or something?